CODEOWNERS

```diff
@@ -0,0 +1,19 @@
+# Owners
+
+SPP Statisitcal Methods Library team
+
+* Neil Smith
```

Comment:
James, there is a github team created called spp-sml (mentioned in the ticket). It is better to just add this team name rather than individuals because then when we add or remove members from the team we do not need to change this file.  Also, the syntax you have used is not correct - read the CODEOWNERS docs for github and you would need to add the github username or email: [codeowners documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)

If you are in doubt about what a file should contain it is always good to look for an example (e.g sml-pythion-small has a [CODEOWNERS file](https://github.com/ONSdigital/sml-python-small/blob/main/CODEOWNERS))

Reply:
done :)